plugin.video.nbcsnliveextra/addon.xml

```diff
@@ -19,6 +19,7 @@
     <forum>http://forum.kodi.tv/showthread.php?tid=207159</forum>
     <website></website>
     <email></email>
+    <broken>This add-on is broken and will no longer be under development.</broken>
```

Comment:
users in helix receiving the update will think the addon will be no longer developed, when in fact it's available from isengard. Can you change this to a more explicit message which suggests them to update their kodi installation?

Reply:
Ah yes, good point.